src/Network/Network/help/Invoke-AzNetworkWatcherNetworkConfigurationDiagnostic.md

```diff
@@ -0,0 +1,344 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
@irrogozh you will need to add a link to the corresponding online help URL here

`https://docs.microsoft.com/en-us/powershell/module/az.network/invoke-aznetworkwatchernetworkconfigurationdiagnostic`

Reply:
Done